Toxygates/src/t/gwt/viewer/client/intermine/InterMineData.java

```diff
@@ -62,7 +62,7 @@ public InterMineData(ImportingScreen parent, @Nullable IntermineInstance preferr
   /**
    * PreferredInstance must not be null
    */
-  public void importLists(final boolean asProbes) {
```

Comment:
Removed this parameter in a number of places since it was always true, and no logic for the false case had ever been implemented.